src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
```

Comment:
Variable?

Reply:
> Variable?

What`s wrong?) I don`t understand) I have used card width a few times in this project, that`s why I put it into the variable. Or you about the name of this variable?

Reply:
```js
border: 1px solid #f3f3f3; <----
```

Reply:
> ```
> border: 1px solid #f3f3f3; <----
> ```

I got it, thanks